maths/greatest_common_divisor.py

```diff
@@ -1,37 +1,26 @@
 """
 Greatest Common Divisor.
 
-Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
+Wikipedia reference: https://en.wikipedia.org/wiki/greatest_common_divisor
```

Comment:
```suggestion
Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
```
The original URL is correct: it's capitalized